cli/cmdlineparser.cpp

```diff
@@ -133,6 +133,9 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
             else if (std::strncmp(argv[i], "--addon=", 8) == 0)
                 mSettings->addons.emplace_back(argv[i]+8);
 
+            else if (std::strncmp(argv[i],"--python=",9) == 0)
+                mSettings->addonPython.assign(argv[i]+9);
```

Comment:
if the path contains a space I guess it would be good to enclose it with `"` or somewhere.

Reply:
Wrapping pythonExe in " when generating command string.